charts/eks/templates/etcd-statefulset.yaml

```diff
@@ -73,6 +73,11 @@ spec:
         - name: data
           emptyDir: {}
       {{- end }}
+      {{- if .Values.init.manifests }}
```

Comment:
I think this is applied in the wrong file, you need to add this to the syncer-deployment